daemons/gptp/common/ieee1588port.cpp

```diff
@@ -145,16 +171,9 @@ bool IEEE1588Port::init_port(int delay[4])
 	this->net_iface = net_iface;
 	this->net_iface->getLinkLayerAddress(&local_addr);
 	clock->setClockIdentity(&local_addr);
+	memcpy(this->link_delay, delay, 4);
```

Comment:
What is the intent here? delay is passed in as a parameter of *int delay[4]*?
I assume link_delay is a single int?